src/server/json_family.cc

```diff
@@ -591,10 +600,10 @@ OpResult<vector<OptBool>> OpToggle(const OpArgs& op_args, string_view key, strin
       }
     };
 
-    OpStatus status = UpdateEntry(op_args, key, path, cb);
-    if (status != OpStatus::OK) {
-      return status;
-    }
+    status = UpdateEntry(op_args, key, path, cb);
+  }
+  if (status != OpStatus::OK) {
+    return status;
   }
   return vec;
```

Comment:
nit ? :

Reply:
ternary operator requires all the returned expressions  be of the same type, which makes it less elegant here